vm/src/obj/objbytearray.rs

```diff
@@ -4,10 +4,10 @@ use crossbeam_utils::atomic::AtomicCell;
 use rustpython_common::borrow::{BorrowedValue, BorrowedValueMut};
 use std::mem::size_of;
 
-use super::objint::PyIntRef;
 use super::objiter;
 use super::objstr::PyStrRef;
 use super::objtype::PyTypeRef;
+use super::{objbytes::PyBytesRef, objint::PyIntRef};
```

Comment:
here also already is `PyBytes` below. I don't mind this kind of grouping when there is nothing in the file, but also want to keep the convention when other codes are already grouped by its own rule.